app/serializers/project_serializer.rb

```diff
@@ -21,12 +21,20 @@ class ProjectSerializer
     subjects_export: { include: false },
     aggregations_export: { include: false }
   can_filter_by :display_name, :slug, :beta_requested, :beta_approved,
-    :launch_requested, :launch_approved, :private, :state
+    :launch_requested, :launch_approved, :private, :state, :live
   can_sort_by :launch_date, :activity, :completeness, :classifiers_count,
     :updated_at, :display_name
 
   def self.page(params = {}, scope = nil, context = {})
-    params["state"] = Project.states[params["state"]] if params.key?("state")
+    if params.key?("state")
+      if Project.states.include?(params["state"])
+        params["state"] = Project.states[params["state"]]
+      else
+        params[params["state"]] = true
```

Comment:
I think this could result in unexpected responses from valid query params, e.g. 
`?launch_approved=false&state=launch_approved` 

This would negate the specific query param resulting in `params[launch_approved]=true`